Transfer/Sources/CreateTransferController.swift

```diff
@@ -0,0 +1,255 @@
+        super.viewDidLoad()
+        title = "transfer_funds".localized()
+        navigationItem.backBarButtonItem = UIBarButtonItem.back
+        largeTitle()
+        setViewBackgroundColor()
+
+        presenter.loadCreateTransfer()
+
+        setUpCreateTransferTableView()
+        hideKeyboardWhenTappedAround()
+    }
+
+    private func setUpCreateTransferTableView() {
+        tableView = UITableView(frame: view.frame, style: .grouped)
+        tableView.accessibilityIdentifier = "createTransferTableView"
+        tableView.cellLayoutMarginsFollowReadableWidth = false
+        registeredCells.forEach {
+            tableView.register($0.type, forCellReuseIdentifier: $0.id)
+        }
+    }
+}
+
+// MARK: - Create transfer table view dataSource
+extension CreateTransferController {
+    override public func numberOfSections(in tableView: UITableView) -> Int {
+        return presenter.sectionData.count
+    }
+
+    override public func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return presenter.sectionData[section].rowCount
+    }
+
+    override public func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        return getCellConfiguration(indexPath)
+    }
+
+    override public func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
+        return presenter.sectionData[section].title
+    }
+
+    override public func tableView(_ tableView: UITableView, titleForFooterInSection section: Int) -> String? {
+        if let transferSection = presenter.sectionData[section] as? CreateTransferSectionTransferData {
+            return transferSection.footer
+        }
+        return nil
+    }
+
+    private func getCellConfiguration(_ indexPath: IndexPath) -> UITableViewCell {
+        let cellIdentifiers = presenter.sectionData[indexPath.section].cellIdentifiers
+        let cell = tableView.dequeueReusableCell(withIdentifier: cellIdentifiers[indexPath.row], for: indexPath)
+        let section = presenter.sectionData[indexPath.section].createTransferSectionHeader
+        switch section {
+        case .destination:
+            getDestinationSectionCellConfiguration(cell, indexPath)
+
+        case .transfer:
+            getTransferSectionCellConfiguration(cell, indexPath)
+
+        case .notes:
+            getNotesSectionCellConfiguration(cell)
+
+        case .button:
+            getButtonSectionCellConfiguration(cell, indexPath)
+        }
+        return cell
+    }
+
+    private func getDestinationSectionCellConfiguration(_ cell: UITableViewCell, _ indexPath: IndexPath) {
+        guard let sectionData = presenter.sectionData[indexPath.section] as? CreateTransferSectionDestinationData,
+            let tableViewCell = cell as? TransferDestinationCell else {
+                return
+        }
+        tableViewCell.accessoryType = .disclosureIndicator
+        if sectionData.isTransferMethodAvailable {
```

Comment:
No need of this additional property isTransferMethodAvailable. You can always do `if let transferMethod = presenter.selectedTransferMethod`. Also make presenter.selectedTransferMethod optional

Reply:
fixed